doc/wiki/Contribute:-Update-Wiki.md

```diff
@@ -1 +1,87 @@
-TBF.
\ No newline at end of file
+We are glad you have something new to contribute to GAMER, and you would like to teach people how to use it!
+
+Before getting started, I recommend you have some basic knowledge of [repository, branch, fork](https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories), [action](https://docs.github.com/en/actions), and [workflow](https://docs.github.com/en/actions/using-workflows).
+
+The followings are the outline of this document:
+* [Setup](#Setup)
+* [Edit Wiki](#Edit-Wiki)
+* [Contribute](#Contribute)
+* [Reference](#Reference)
+
+## Setup
+1. **Check GAMER version**
+   * Please make sure your current GAMER commit is later than commit `13409ab33b12d84780076b6a9beb07317ca145f1` (committed on 2024/06/10).
+
+1. **Create first page**
+   * Click the Wiki page and create the first page
+
+   [[images/CreateWiki.png]]
+
+1. **Create a token for the action bot**
+   * Go to `Setting` of your account > `Developer setting` > `Personal access tokens` > `Generate new token`
+   * Please check the `repo` and the `workflow` options.
+   * You might want to set the `Expiration` to `No expiration`.
+
+   [[images/CreateToken.png]]
+
+   * Remember to save the token since it will only be shown once!
+
+1. **Create a repository secret token**
+   * Go to `Setting` of the repository > `Security` > `Secrets and variables` > `Actions` > `Repository secrets`
+   
+   [[images/CreateSecret.png]]
+
+   * Click `New repository secrets`, and then you will see the following. Please replace `<your personal access token>` to your token at `Secret` and make sure the `Name` of the token is `TOKEN_FOR_WIKI`. 
+
+   [[images/SetToken.png]]
+
+1. **Enable actions**
+   * Enable workflows to run in the forked repository.
+   * Click `Actions` > click the green button.
+   ![image](https://github.com/ChunYen-Chen/CheckNode/assets/70311975/9e58d4a8-3248-4ceb-81ff-276a6943149d)
+
+1. **Initialize wiki**
+   * Click `Action` > `Workflows` > `Copy doc to wiki` > `Run workflow` > choose `Branch: master` > Click green `Run workflow`. Once the workflow is done, the wiki is also updated to the master branch version.
+
+   [[images/InitializeWiki.png]]
+
+## Edit Wiki
+We provide two examples of editing the wiki pages: from GitHub website and through `gollum`. In the following examples, we call the new branch you would like to contribute as `new_contribution_branch`.
```

Comment:
Some old-fashioned people (like me) may want to edit the doc in the local terminal like the other source code. Maybe you can mention we can still use the `Copy doc to wiki` to preview the wiki once the changes in `new_contribution_branch` have been pushed to GitHub.

Reply:
I have added a new example to this section.